edgexfoundry/app-rfid-llrp-inventory
# feat!: updates for common config

BREAKING CHANGE: configuration file changed to remove common config settings







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-rfid-llrp-inventory/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
run edgex non-secure stack 
build service from this branch
run locally with `-cp` option
Verify bootstraps w/o error
Verify log level can be changed from Consul (ignore first error which is fixed in later bootstrap)

## New Dependency Instructions (If applicable)
